lib/etcher.js

```diff
@@ -19,12 +19,47 @@
 const electron = require('electron');
 const path = require('path');
 const elevate = require('./elevate');
+const fs = require('fs');
+const args = require('minimist')(process.argv.slice(2));
 let mainWindow = null;
 
 electron.app.on('window-all-closed', electron.app.quit);
 
 electron.app.on('ready', function() {
 
+  if (args.h || args.help) {
+    console.log('Etcher - A better way to flash OS images to SD cards');
```

Comment:
`yards` "usage" feature could help a lot here. Also, by declaring the accepted options, the "Options" section can be automatically generated, which would help having the options and their documentation in sync.